link_shortener/templates.py

```diff
@@ -2,6 +2,8 @@
 Copyright (C) 2020 Link Shortener Authors (see AUTHORS in Documentation).
 Licensed under the MIT (Expat) License (see LICENSE in Documentation).
 '''
+from decouple import config
+
 from jinja2 import Environment, PackageLoader
```

Comment:
Technicky stejná skupina importů, obojí jsou knihovny. Ale to je jen nitpick

Reply:
9ec275cf79802b1c4da9f79082e9df7f31cf8484